blueocean-dashboard/src/main/js/components/PipelineRunGraph.jsx

```diff
@@ -47,14 +67,20 @@ function convertJenkinsNodeDetails(jenkinsNode, isCompleted) {
         state = 'not_built';
         completePercent = 0;
     }
+    const i18nDuration = timeManager.format(durationMillis, translate('common.date.duration.hint.format', { defaultValue: 'M [month], d [days], h[h], m[m], s[s]' }));
+
+    const title = translate(`common.state.${state}`, { 0: i18nDuration });
 
-    return {
+    const converted = {
         name: jenkinsNode.displayName,
         children: [],
         state,
         completePercent,
         id: jenkinsNode.id,
+        title,
     };
+    logger.debug('converted node', converted);
```

Comment:
Sorry dude .... I know it's debug level but I really do think this kind of logging is only going to be of use to yourself (and maybe not even you once you forget the context in a few weeks) => making it just meaningless/irritating noise for everyone else.

I do appreciate that it's not easy to get the right balance on this stuff. We need to think of this logging as a valuable runtime/production tool that everyone can use i.e. something that helps Jenkins users to help us diagnose issues as they are reported etc, and not just a tool for us to use while we're developing the initial code.

Don't get me wrong bud ... I love the way you take things and run with them ... this is just a NIT 😄  and I'm not trying to be a PITA !!!!